include/CppUTestExt/GTestConvertor.h

```diff
@@ -247,45 +244,49 @@ class GTestResultReporter : public ::testing::ScopedFakeTestPartResultReporter
     }
 };
 
-inline GTestShell::GTestShell(::testing::TestInfo* testinfo, GTestShell* next, GTestFlagsThatAllocateMemory* flags) : testinfo_(testinfo), next_(next), flags_(flags)
+inline GTestShell::GTestShell(::testing::TestInfo* testinfo, GTestShell* next, GTestFlagsThatAllocateMemory* flags) :
+    testinfo_(testinfo),
+    next_(next),
+    flags_(flags)
 {
     setGroupName(testinfo->test_case_name());
     setTestName(testinfo->name());
 }
 
-class GTestUTest: public Utest {
+class GTestUTest : public Utest
+{
 public:
-    GTestUTest(::testing::TestInfo* testinfo, GTestFlagsThatAllocateMemory* flags) : testinfo_(testinfo), test_(NULL), flags_(flags)
+    GTestUTest(::testing::TestInfo* testinfo, GTestFlagsThatAllocateMemory* flags) :
+        testinfo_(testinfo),
+        test_(NULL),
+        flags_(flags)
     {
-
     }
 
     void testBody()
     {
         try {
             test_->TestBody();
         }
-        catch (CppUTestFailedException& ex)
-        {
+        catch (CppUTestFailedException& ex) {
```

Comment:
TODO: break between `catch` and brace

Reply:
This does not appear possible. Clang-format lumps `catch` in with the other controls statements (`if`, `for`, etc.).